seek-n-major.json

```
@@ -0,0 +1,135 @@
+{
+  "extends": [
+    ":prHourlyLimit4",
+    ":rebaseStalePrs",
+    ":renovatePrefix",
+    ":semanticCommits",
+    ":timezone(Australia/Melbourne)",
+    ":updateNotScheduled",
+    "preview:buildkite",
+    "preview:dockerCompose",
+    "docker:disableMajor"
+  ],
+  "packageRules": [
+    {
+      "depTypeList": ["dependencies"],
+      "semanticCommitType": "fix"
+    },
+    {
+      "digest": {
+        "commitMessageExtra": "",
+        "groupName": "gomod digests",
+        "schedule": "after 3am and before 6am on the first day of the month"
+      },
+      "matchManagers": ["gomod"],
+      "semanticCommitType": "fix"
+    },
+    {
+      "commitMessageExtra": "{{newValue}}",
+      "commitMessageTopic": "{{depName}}",
+      "matchManagers": ["buildkite", "gomod", "npm", "nvm"]
+    },
+    {
+      "matchManagers": ["npm"],
+      "matchPackageNames": ["@types/node"],
+      "matchUpdateTypes": ["major"],
+      "enabled": false
+    },
+    {
+      "automerge": true,
+      "commitMessageExtra": "",
+      "groupName": "npm definitely typed",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@types/"],
+      "prPriority": 99,
+      "recreateClosed": true,
```

I'd try ungrouping these; majors usually require a bit more work and it's handy to be able to triage and possibly ignore them individually.

```suggestion
      "matchManagers": ["npm"],
      "matchPackagePatterns": ["^@types/"],
      "prPriority": 99,
```